### PR TITLE
minor refactoring for smaller, simpler (?) code

### PR DESCRIPTION
**Pull request by 2is10**
cc @stkem

### CODE REVIEW

File: kifi-backend/modules/shoebox/public/js/track.js [hunk 1]
```diff
@@ -1,68 +1,45 @@
-/**
- * ---------------
- *      Track
- * ---------------
- *
- * Contains functionality to track user interactions on the Site.
- * It only requires the mixpanel tag to be on the page (+jquery),
- * Everything else is stand alone (i.e. it attaches itself to the elements)
- */
+// Tracks user actions using mixpanel. Also requires jQuery.
```
Comment (2is10) [inline kifi-backend/modules/shoebox/public/js/track.js:1]: smaller comment since we don’t yet minify

**Comment by 2is10** [comment]
Let me know if anything looks bad and I’ll quickly change it.

**Comment by stkem** [comment]
Dude, I get the need for smaller code but please don't change 70+% of lines (including variable names!) in code I'm actively working on in the future.

**Comment by 2is10** [comment]
Sorry. Mentioned I’d be quickly changing some things. I thought you were
done for a little while.​

**Pull request merged**